packages/contracts/contracts/foundry/SphinxConstants.sol

```diff
@@ -9,4 +11,175 @@ contract SphinxConstants {
   address public constant safeFactoryAddress = 0xa6B71E26C5e0845f74c812102Ca7114b6a896AB2;
   address public constant safeSingletonAddress = 0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552;
   address public constant sphinxModuleImplAddress = 0x8f4E4d51B8050B0ff713eff1F88f3dD8b5e8a530;
+
+  uint8 internal constant numSupportedNetworks = 23;
+
+  function getNetworkInfoArray() public pure returns (NetworkInfo[] memory) {
```

Comment:
We now automatically generate this function and the Network enum from the network array defined in typescript.